docs/iceberg-rest-service.md

```diff
@@ -107,11 +107,12 @@ Please refer to [Credential vending](./security/credential-vending.md) for more
 
 #### S3 configuration
```

Comment:
Could you change the Iceberg catalog document in https://github.com/apache/gravitino/blob/main/docs/lakehouse-iceberg-catalog.md#L68 and Spark document in https://github.com/apache/gravitino/blob/main/docs/spark-connector/spark-catalog-iceberg.md#L100 too?